src/datasources/asset/components/AssetQueryUtilizationEditor.tsx

```diff
@@ -0,0 +1,170 @@
+    }
+  };
+  const handleEntityTypeChange = (value: EntityType): void => {
+    if (query.entityType !== value) {
+      common.handleQueryChange({ ...query, entityType: value }, true);
+    }
+  };
+  const loadAssetIdOptions = (): Array<SelectableValue<string>> => {
+    let options: SelectableValue[] = (assetIds.value ?? []).map((asset: AssetModel): SelectableValue<string> => ({
+        'label': asset.name,
+        'value': asset.id,
+        description: asset.id
+      })
+    );
+    options.unshift(...common.getVariableOptions());
+
+    return options;
+  }
+
+  return (
+    <>
+      <InlineField label={"Calculate utilization for"}
+                   tooltip={tooltips.entityType} labelWidth={22}>
+        <RadioButtonGroup
+          options={entityTypeOptions}
+          value={query.entityType}
+          onChange={(item: any) => handleEntityTypeChange(item)}
+        />
+      </InlineField>
+      <InlineField label="Workspace" labelWidth={22} tooltip={'The workspace to search for the asset specified.'}>
+        <Select
+          isClearable
+          isLoading={workspaces.loading}
+          onChange={onWorkspaceChange}
+          options={workspaces.value}
+          placeholder="Any workspace"
+          value={query.workspace}
+        />
+      </InlineField>
+      <InlineField label="System"
+                   tooltip={tooltips.system[query.entityType]}
+                   labelWidth={22}>
+        <MultiSelect
+          isClearable
+          allowCreateWhileLoading
+          options={common.loadMinionIdOptions(minionIds)}
+          isValidNewOption={isValidId}
+          onChange={handleMinionIdsChange}
+          placeholder="Select systems"
+          width={85}
+          value={query.minionIds.map(toOption)}
+        />
+      </InlineField>
+      {query.entityType === EntityType.Asset && (
+        <InlineField label="Asset Identifier"
+                     tooltip={tooltips.assetIdentifiers}
+                     labelWidth={22}>
+          <MultiSelect
+            width={85}
+            allowCreateWhileLoading
+            isClearable={true}
+            isValidNewOption={isValidId}
+            options={loadAssetIdOptions()}
+            onChange={handleAssetIdentifierChange}
+            placeholder="Search by name or enter id"
+            value={query.assetIdentifiers.map(toOption)}
+          />
+        </InlineField>
+      )}
+      <FloatingError message={errorMsg}/>
+    </>
+  )
+}
+
+const tooltips = {
+  entityType: `Calculate utilization for one or more systems or assets.`,
+
+  workspace: {
+    [EntityType.Asset]: `The workspace where you want to search for the assets.`,
+    [EntityType.System]: `The workspace where you want to search for the systems.`,
+  },
+
+  assetIdentifiers: 'Search assets by name or enter id.',
+
+  system: {
+    [EntityType.Asset]: `Filter assets by system.`,
+    [EntityType.System]: `Search systems by name or enter an ID`,
+  },
+};
```

Comment:
@vpstoynova Can you review these strings?

@TigranVardanyan Can you provide a screenshot for Vera showing how these will appear in the product?

Reply:
@mure provided

Reply:
```suggestion
const tooltips = {
  entityType: `Calculate utilization for one or more systems or assets.`,

  workspace: {
    [EntityType.Asset]: `The workspace where you want to search for the assets.`,
    [EntityType.System]: `The workspace where you want to search for the systems.`,
  },

  assetIdentifiers: 'Search assets by name or enter an ID.',

  system: {
    [EntityType.Asset]: `Filter assets by system.`,
    [EntityType.System]: `Search systems by name or enter an ID.`,
  },
};
```